_posts/2016/2016-05-13-geecon.md

```diff
@@ -0,0 +1,86 @@
+
+Here is a list of my personal highlights:
+
+## Josh Long - "Cloud Native Java"
+
+Josh Long ([@starbuxman](https://twitter.com/starbuxman)) is the spring developer advocate that travels the world to spread the word about the greatness of spring boot to implement *Cloud Native Applications*. This is another - maybe better suited term - for microservices. At epages we work a lot with Spring Cloud and Spring Boot so most of the content was not new to me. The interesting part was rather that Josh is the master of live-coding. He does that at an incredible speed and with a great level of entertainment. So that was really a show.
+
+> <blockquote class="twitter-tweet" data-lang="de"><p lang="en" dir="ltr">Discovered a few killer features in Spring Boot thanks to <a href="https://twitter.com/starbuxman">@starbuxman</a> at <a href="https://twitter.com/hashtag/geecon?src=hash">#geecon</a> ;-) <a href="https://t.co/jJiAzF739m">pic.twitter.com/jJiAzF739m</a></p>&mdash; Piotr Wittchen (@piotr_wittchen) <a href="https://twitter.com/piotr_wittchen/status/730421205967228929">11. Mai 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Daniel Sawano and Daniel Deogun - "Beyond lambdas - the aftermatch"
+
+The two Daniels ([@DanielSawano](https://twitter.com/DanielSawano), [@DanielDeogun](https://twitter.com/DanielDeogun)) focussed their talk on lambda expressions introduced in Java 8. In my team we love this language construct and use it extensively. The talk showed some common problems, smells and pitfalls when using lambdas:
+
+- things that are obvious for functional programmers can be hard to adapt to for java developers and you need to be very well  aware of what gets evaluated and when - so you better know the difference between `Optional.orElse` and `Optional.orElseGet`
+- the compact and fluent syntax of lambdas can hide issues where the separation of concerns is violated
+
+I liked the talk because the topic was specific and small so it was possible to dive rather deep. Also they presented it in a really nice way by illustrating their points with live-coding and telling the whole story as a dialogue between two developers.
+
+> <blockquote class="twitter-tweet" data-lang="de"><p lang="en" dir="ltr"><a href="https://twitter.com/DanielSawano">@DanielSawano</a> <a href="https://twitter.com/DanielDeogun">@DanielDeogun</a> great talk about pitfalls and best-practices with Java8 Lambdas - and fun too 💯 <a href="https://twitter.com/hashtag/geecon?src=hash">#geecon</a> <a href="https://t.co/gWpkvIozLK">pic.twitter.com/gWpkvIozLK</a></p>&mdash; Mathias D (@zaddo) <a href="https://twitter.com/zaddo/status/730666445688979456">12. Mai 2016</a></blockquote>
```

Comment:
The Twitter pics are implemented as hrefs and are displayed as comment.
Better implement it as image next to the text, left or right, e.g. like that:
{% image blog/imagename.jpg 40% left %} {% endimage %}